xilinx/common/tcl/prologue.tcl

```diff
@@ -26,6 +26,9 @@ while {[llength $argv]} {
     -post-impl-debug-tcl {
       set argv [lassign $argv[set argv {}] post_impl_debug_tcl]
     }
+    -env-var-srcs {
```

Comment:
Please add a comment describing what exactly this argument is and how it works, because this is admittedly a kind of roundabout way of doing this, which is not immediately obvious from reading the code. I only know because we had the discussion in person yesterday, but the two levels of resolving values is unusual enough that I think it warrants an explicit comment.